src/main/java/bio/terra/service/tabulardata/google/BigQueryProject.java

```diff
@@ -97,9 +99,28 @@ public boolean deleteDataset(String datasetName) {
         }
     }
 
-    public void createTable(String datasetName, String tableName, Schema schema) {
+    public void createUnpartitionedTable(String datasetName, String tableName, Schema schema) {
+        createTable(datasetName, tableName, schema, null, null);
+    }
+
+    public void createTimePartitionedTable(String datasetName, String tableName,
+                                           Schema schema, TimePartitioning timePartitioning) {
+        createTable(datasetName, tableName, schema, timePartitioning, null);
+    }
+
+    public void createRangePartitionedTable(String datasetName, String tableName,
+                                            Schema schema, RangePartitioning rangePartitioning) {
+        createTable(datasetName, tableName, schema, null, rangePartitioning);
+    }
+
+    private void createTable(String datasetName, String tableName, Schema schema,
+                             TimePartitioning timePartitioning, RangePartitioning rangePartitioning) {
```

Comment:
Again, this smells like a good case for a subclass structure with a create method.